Dockerfile.test-infra

```diff
@@ -10,8 +10,9 @@ RUN curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.10.1/m
   && chmod +x minikube && mkdir -p /usr/local/bin/ && install minikube /usr/local/bin/
 RUN curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/bin/
 
-RUN mkdir -p ~/.terraform.d/plugins
-RUN curl -SL https://github.com/dmacvicar/terraform-provider-libvirt/releases/download/v0.6.2/terraform-provider-libvirt-0.6.2+git.1585292411.8cbe9ad0.Fedora_28.x86_64.tar.gz | tar -xz -C ~/.terraform.d/plugins/
+RUN mkdir -p /root/.terraform.d/plugins
```

Comment:
mm root inside the container?

Reply:
It's already been referred on the Makefile under `_run_terraform`.
It's just a fix to support `podman` that builds "Rootless Builds" and `~/` might not indicate `/root/`